### PR TITLE
refactor dissimilarity function from omni to mds

### LINKED ISSUE #48: Refactor dissimilarity function from omni to cmds
title

### PR DESCRIPTION
**Pull request by j1c**
Closes #48 

Refactor the computing of dissimilarity to ClassicalMDS from omni. Looking for feedback for explanation for n_elements if it makes any sense. Generalized it so that it can compute row-wise distance or block-wise distance.

### CODE REVIEW

File: graspy/embed/mds.py [hunk 1]
```diff
@@ -51,63 +60,152 @@ class ClassicalMDS(BaseEstimator):
...
+
+        Parameters
+        ----------
+        X : array, shape (n_samples, n_features) or (n_samples, n_samples)
+            Input data. If ``dissimilarity=='precomputed'``, the input should
+            be the dissimilarity matrix.
+        
+        n_elements : int, optional, default: 1
+            If n_elements > 1, it computes the pairwise euclidean distance between
+            matrices given by (n_elements, n_features). n_elements must divide evenly
```
Comment (bdpedigo) [inline graspy/embed/mds.py]: about the n_elements explanation - when you say the matrices are given by (n_elements, n_features) do you mean that is the dimension of each resulting matrix that it compares?

Comment (j1c) [reply graspy/embed/mds.py]: Yea. So if n_elements=5, and your input X is 100 by 2 matrix, then it takes the first 5 rows as one sample matrix and computes distance to the second 5 rows, etc. Still not sure how to word this clearly.

Comment (j1c) [reply graspy/embed/mds.py]: Another approach is to allow for any n-dimensional arrays and compute distance along axis=0. So for outputs of omni, reshape it such that it has shape (n_graphs, n_vertices, embedded_dimensions).

**Comment by j1c** [comment]
Computing dissimilarity just takes in any array >=2d, assuming first axis refers to the `n_samples`. 

Also added a nice reference.

**Pull request merged**